src/libANGLE/renderer/d3d/d3d9/TextureStorage9.cpp

```diff
@@ -92,6 +92,20 @@ int TextureStorage9::getLevelCount() const
     return static_cast<int>(mMipLevels) - mTopLevel;
 }
 
+int TextureStorage9::getLevelWidth(int mipLevel) const
+{
+    return std::max(static_cast<int>(mTextureWidth) >> mipLevel, 1);
+}
+
+int TextureStorage9::getLevelHeight(int mipLevel) const
+{
+    return std::max(static_cast<int>(mTextureHeight) >> mipLevel, 1);
+}
+int TextureStorage9::getLevelDepth(int mipLevel) const
```

Comment:
nit^2: unused parameter name can be omitted

Reply:
convention in the codebase apparently is to keep them